apollo-router/Cargo.toml

```diff
@@ -67,6 +67,7 @@ anyhow = "1.0.80"
 apollo-compiler = "=1.0.0-beta.14"
 apollo-federation = "=0.0.9"
 arc-swap = "1.6.0"
+async-channel = "1.9.0"
```

Comment:
Could you use [broadcast channels](https://docs.rs/tokio/latest/tokio/sync/broadcast/index.html) from tokio instead ? As we're already using it for subscription and it's already in our deps

Reply:
NVM it won't work